Description: make the error message more helpful on Ubuntu
Forwarded: no
Updated: 2014-02-17

Index: virt-manager-1.3.2/virtManager/engine.py
===================================================================
--- virt-manager-1.3.2.orig/virtManager/engine.py	2016-03-31 17:43:53.351722160 -0400
+++ virt-manager-1.3.2/virtManager/engine.py	2016-03-31 17:43:53.351722160 -0400
@@ -672,7 +672,10 @@
                           "regular user. Try running as root.")
                 show_errmsg = False
             elif re.search(r"libvirt-sock", tb):
-                hint += _("Verify that the 'libvirtd' daemon is running.")
+                hint += _("Verify that:\n" +
+                         " - The 'libvirt-bin' package is installed\n" +
+                         " - The 'libvirtd' daemon has been started" +
+                         (conn.is_qemu_system() and "\n - You are member of the 'libvirtd' group" or ''))
                 show_errmsg = False
 
         probe_connection = self.conns[conn.get_uri()]["probeConnection"]
